meta-resin-common/recipes-support/resin-logs/resin-logs/resin-logs

```diff
@@ -4,12 +4,21 @@ set -e
 
 . /usr/sbin/resin-vars
 
-if [ "$PERSISTENT_LOGGING" == "true" ]; then
-    echo "[INFO] Configure persistent logs in /mnt/conf/root-overlay/var/log/journal"
-    echo -e "[Journal]\nStorage=persistent\n" > /etc/systemd/journald.conf.d/resin-logs.conf
-elif [ "$PERSISTENT_LOGGING" == "false" ]; then
-    echo "[INFO] Configure volatile logs in /run/log/journal"
-    echo -e "[Journal]\nStorage=volatile\n" > /etc/systemd/journald.conf.d/resin-logs.conf
+if [ "$PERSISTENT_LOGGING" == "true" ] && [ ! -d /var/log/journal ]; then
+    systemctl start var-log-journal.mount
+    # Send flush signal to journald
+    killall -USR1 systemd-journald
+elif [ -d /var/log/journal ]; then
+    systemctl stop systemd-journald-dev-log.socket
+    systemctl stop systemd-journald.socket
+    systemctl stop systemd-journald.service
+    systemctl stop var-log-journal.mount
+
+    # Copy persistent logs to ram to keep them consistent
+    cp -r /var/log/journal /run/log ; rmdir /var/log/journal
```

Comment:
Curious, why not "mv"? Does that comment contain the explanation? If so, I don't understand.